pages/linux/flock.md

```diff
@@ -0,0 +1,16 @@
+# flock
+
+> Manage locks from shell scripts.
+> You can use it to ensure that only one progress of your command is running.
```

Comment:
```suggestion
> It can be used to ensure that only one progress of a command is running.
```
We try to avoid personal pronouns. Also, did you mean to write `process` instead of `progress`?